### ISSUE TITLE
Setting up stop loss trigger and stop loss price with an order for okx dosent works.

### ISSUE THREAD

**Issue by Sailendrakc** [ccxt/ccxt#18108]
### Operating System

win 10

### CCXT Version

3.0.78

### Description

Hello guyz,

I am trying to place a long order with stop loss price and stoploss trigger price associated with that order. The order is created sucessfully but stoploss details are ignored.

I went through this post as well and got the idea of using unified stoploss aproach (3rd option):
https://github.com/ccxt/ccxt/issues/13822#issuecomment-1158682202

 You can copy paste the below code and run in localhost to replicate. Any info on this issue would be useful. Thank you!

### Code

```
  <!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    
</body>
<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/ccxt@3.0.78/dist/ccxt.browser.js"></script>
<script async>
(async function(){
    
    let exName = "okx"
    let ccxtpro = ccxt.pro;
    let exchange = new ccxtpro[exName]();

    //auth details
    let details = {
        "okx": {
            apiKey: 'xxxxxxxxxxxxxxxxxx',
            secret: 'xxxxxxxxxxxxxxxxxx',
            uid: 'xxxxxxxxx',
            password: 'xxxxxxxxxx'
        }
    }

    //perform auth
    let dt = details['okx'];
    if (dt) {
        exchange.apiKey = dt.apiKey;
        exchange.secret = dt.secret;
        exchange.uid = dt.uid;
        exchange.password = dt.password;
        exchange.nonce = function () {
        return exchange.milliseconds();
            }
    }

    //load markets
    await exchange.loadMarkets();
    let symbol = "DOT-USDT-SWAP";
    
    //get the latest price and go long on 20 percent less price.
    let rate = await exchange.fetch_ticker(symbol, null);
    rate = rate.bid - (0.20 * rate.bid);

    //stop loss is 10 percent lower amount than the buying rate
    let stoploss = rate - (0.10 * rate);

    //stop loss trigger is 5 percent above than actual stop loss
    let stoplossTrigger = stoploss + (0.05 * stoploss);

    let params = {
            'stopLoss': {
                'type': 'limit', // or 'market'
                'price': stoploss,
                'triggerPrice': stoplossTrigger,
            },
            'posSide': 'long',
    }

    let type = 'limit';
    let side = "buy";
    let amount = 1;

    //set leverage
    let lev = await exchange.setLeverage(1, symbol, {});

    //create long order
    let id = await exchange.createOrder(symbol = symbol, type = type, side = side, amount = amount, rate = rate, params);     

    //fetch the order detail to check if stoploss has been setup or no.
    let res = await exchange.fetch_open_orders(symbol, undefined, 5, {});
    console.log(res);

    //we can see stoploss prices are undefined.

})();

</script>
</html>

```

**Comment by sc0Vu** [comment]
Hi @Sailendrakc 

Sorry for your inconvenience, ccxt call algo order when both sl/tp price were set in `createOrder` for now. To attach sl/tp price on the position, we should call another one. Anyway, we'll update this.
